easybuild/easyconfigs/a/AutoDock/AutoDock-4.2.6-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,30 @@
+# Currently there is not an EasyBlock to unify AutoDock and AutoGrid so this is a happy medium until then
+easyblock = 'ConfigureMake'
+
+name = 'AutoDock'
+version = '4.2.6'
+
+homepage = 'https://autodock.scripps.edu/'
+
+description = """
+ AutoDock is a suite of automated docking tools. It is designed to 
+ predict how small molecules, such as substrates or drug candidates, bind to 
+ a receptor of known 3D structure.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
```

Comment:
@JackPerdue Why change this from `GCC` to `GCCcore` (compared to the existing easyconfigs).

`GCC` is equivalent to `iccifort`, while `GCCcore` is a base for both (and makes most sense to use for dependencies that are not performance-sensitive).

Reply:
What if I want to use AutoDock as a dependency in a intel-2019b build later?  GCCcore will work with minimal-toolchains.  Last I checked, GCC won't (since GCC isn't part of the intel toolchain)

Reply:
For that we create a `AutoDock-4.2.6-iccifort-....eb` variant as well, and build both.

Reply:
And does the Intel compilers significantly outperform GCC in AutoDock/Grid?  Why build two when one will do?

Reply:
The general policy has been to always err on the possibility that intel performs better (because it quite often actually does). Since shadowing modules is a bit iffy, the question has been simply been the reverse: "why prevent an intel version when we don't know it doesn't perform better?". In many cases, it actually does perform better.

Disclaimer: Please view my input as simply explanation. I have no strong feelings regarding autodock.

Reply:
There is nothing that precludes you from using the iccifort version.  Just don't set EASYBUILD_MINIMAL_TOOLCHAINS (while possibly using EASYBUILD_USE_EXISTING_MODULES) and your iccifort version will be used.

I see no reason to have to build for Intel/PGI/GCC when I can just build with GCCcore and use it on all three (unless there are demonstrable/provable performance benefits...)